QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeBase.java

```diff
@@ -51,6 +53,16 @@ protected void collectTote() throws AutonomousModeOverException,
 		waitUntil(BooleanMixing
 				.invert((BooleanInputPoll) QuasarHelios.autoLoader));
 	}
+	
+	protected void setClamp(boolean value) throws InterruptedException, AutonomousModeOverException {
+		QuasarHelios.clamp.openControl.set(value);
+		waitForTime(30l);
+	}
```

Comment:
Might want to rename this to make it clear that it's talking about opening/closing.